src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapGroupBySettings.jsx

```diff
@@ -1,20 +1,23 @@
-import React from 'react';
```

Comment:
Martin, do you develop with the UI open as a workspace in VsCode, so that the code is auto-formatted and corrected after save? I just checked out your changes locally and this file looks full of linting errors (commas, formatting, etc.)

Reply:
I do, in the linux computer I got the auto formatting, but here it seems to not be working.

I'll try to make it work and fix this.

Reply:
Thanks Martin, the auto formatting is super helpful, make sure that you have it working.